src/chart/generateCategoricalChart.tsx

```diff
@@ -1634,7 +1602,7 @@ export const generateCategoricalChart = ({
 
     handleTouchMove = (e: React.TouchEvent) => {
       if (e.changedTouches != null && e.changedTouches.length > 0) {
-        this.handleMouseMove(e.changedTouches[0]);
+        this.throttleTriggeredAfterMouseMove(e.changedTouches[0]);
```

Comment:
aren't these two functions slightly different? Missing e.persist now

Reply:
same!

Reply:
Come to think of it, there's no 'persist' for touch events, right? It was never there originally

Reply:
I think it doesn't matter the type of event - all synthetic events need e.persist (react < 17 at least)

https://legacy.reactjs.org/docs/legacy-event-pooling.html

Reply:
I know. But we are not passing events here